test/lib/fs.readdir.spec.js

```diff
@@ -83,6 +91,29 @@ describe('fs.readdir(path, callback)', function() {
     );
   });
 
+  it('calls with an error for restricted path', function(done) {
+    fs.readdir('denied', function(err, items) {
+      assert.instanceOf(err, Error);
```

Comment:
Suggest to add one line on the error code.
```js
assert.equal(err.code, 'EACCES');
```

Reply:
Good idea: done!